src/library_webgl.js

```diff
@@ -1362,19 +1362,19 @@ var LibraryGL = {
   },
 
   glGetIntegerv__sig: 'vii',
-  glGetIntegerv__deps: ['$emscriptenWebGLGet'],
+  glGetIntegerv__deps: ['$emscriptenWebGLGet', '$writeI53ToI64'],
   glGetIntegerv: function(name_, p) {
     emscriptenWebGLGet(name_, p, {{{ cDefine('EM_FUNC_SIG_PARAM_I') }}});
   },
 
   glGetFloatv__sig: 'vii',
-  glGetFloatv__deps: ['$emscriptenWebGLGet'],
+  glGetFloatv__deps: ['$emscriptenWebGLGet', '$writeI53ToI64'],
```

Comment:
why is there a new dependency in this function, it just calls `emscriptenWebGLGet` which is already in the deps for it? If `emscriptenWebGLGet` needs that function, should it be in the deps for that function?

Reply:
Thanks, good catch. When I was writing this, I misinterpreted that `emscriptenWebGLGet` would have lived off of the `GL` object and would not be able to add deps to it directly, but of course it is now a self-standing function so that's not the case anymore. Fixed.

Reply:
I still see the extra `$writeI53ToI64` deps?

Reply:
Err, thanks, I had forgot to push!